### PR TITLE
automatically check if the response from /filters/attributes is aligned with API specification

### PR DESCRIPTION
**Pull request by kjaskiewiczz**

### CODE REVIEW

File: tests/tests/client.py [hunk 1]
```diff
@@ -147,6 +147,33 @@ def addDeviceToGroup(self, group, device, expected_error=False, JWT="foo.bar.baz
...
+        }
+
+        http_client = RequestsClient()
+        http_client.session.verify = False
+
+        self.client = SwaggerClient.from_spec(
+            load_file(spec), config=config, http_client=http_client,
+        )
+        self.client.swagger_spec.api_url = (
+            "http://%s/api/management/v2/inventory/" % host
```
Comment (merlin-northern) [inline tests/tests/client.py:169]: nit pick dont we have it defined in some cost somewhere? (I have not checked)

**Review by merlin-northern — APPROVED** [review]
lgtm

File: tests/tests/client.py (hunk 1 above)
Comment (kjaskiewiczz) [reply tests/tests/client.py:169]: with openapi it's completely different, but bravado is a bit clunky, and this is what we do in other places where we need v2 API

File: tests/tests/test_mgmt_v2_filters_attributes.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        )
+        attributeList.append(attr)
+
+        did = "some-device-id"
+        internal_client.create_device(did, attributeList)
+        res = management_client_v2.getFiltersAttributes()
+        for attr in res:
+            if attr.scope == "inventory":
+                assert res[0].name == "foo"
+                assert res[0].count == 1
```
[the remark this answers is not in the document]
Comment (mzedel) [reply tests/tests/test_mgmt_v2_filters_attributes.py:50]: Thanks for the clarification offline @kjaskiewiczz !

**Pull request merged**